code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -516,3 +516,73 @@
 		human_user.adjustStaminaLoss(-10, FALSE)
 		human_user.adjustToxLoss(-10, FALSE)
 		human_user.adjustOxyLoss(-10)
+
+/obj/effect/proc_holder/spell/pointed/manse_link
+	name = "Mansus Link"
+	desc = "Piercing through reality, connecting minds. This spell allows you to add people to a mansus net, allowing them to communicate with eachother"
+	school = "transmutation"
+	charge_max = 600
```

Comment:
don't really see why this needs a long cooldown; this is just gonna result in constructs waiting around near the raw prophet for ages before they can be sent out